packages/core/scripts/serverless-orchestration/ServerlessHub.js

```diff
@@ -137,25 +137,42 @@ hub.post("/", async (req, res) => {
     await delay(2); // Wait a few seconds to be sure the the winston logs are processed upstream.
     res.status(200).send({ message: "All calls returned correctly", output: { errorOutputs, validOutputs } });
   } catch (errorOutput) {
-    logger.debug({
-      at: "ServerlessHub",
-      message: "Some spoke calls returned errors (details)🚨",
-      output: errorOutput instanceof Error ? errorOutput.message : errorOutput
-    });
-    logger.error({
-      at: "ServerlessHub",
-      message: "Some spoke calls returned errors 🚨",
-      output:
-        errorOutput instanceof Error
-          ? errorOutput.message
-          : {
-              errorOutputs: Object.keys(errorOutput.errorOutputs), // eslint-disable-line indent
-              validOutputs: Object.keys(errorOutput.validOutputs) // eslint-disable-line indent
-            } // eslint-disable-line indent
-    });
+    // If the errorOutput is an instance of Error then we know that error was produced within the hub.
+    if (errorOutput instanceof Error) {
+      logger.error({
+        at: "ServerlessHub",
+        message: "A fatal error occurred in the hub",
+        output: errorOutput.message
+      });
+    } else {
+      // Else, the error was produced within one of the spokes. If this is the case then we need to process the errors a bit.
+      logger.debug({
+        at: "ServerlessHub",
+        message: "Some spoke calls returned errors (details)🚨",
+        output: errorOutput
+      });
+      logger.error({
+        at: "ServerlessHub",
+        message: "Some spoke calls returned errors 🚨",
+        errorOutputs: Object.keys(errorOutput.errorOutputs).map(spokeName => {
+          try {
+            return {
+              spokeName: spokeName,
+              errorReported: errorOutput.errorOutputs[spokeName].execResponse
+                ? errorOutput.errorOutputs[spokeName].execResponse.stderr
+                : JSON.stringify(errorOutput.errorOutputs[spokeName])
```

Comment:
Should we worry about edge case where `errorOutput.errorOutputs[spokeName]` is larger than SlackTransport can handle?

Reply:
I think the SlackTransport should handle this. Can leave it for https://github.com/UMAprotocol/protocol/issues/2121